rmw_opensplice_cpp/src/types.cpp

```diff
@@ -342,3 +345,105 @@ size_t OpenSpliceSubscriberListener::current_count() const
 {
   return current_count_;
 }
+
+rmw_ret_t OpenSpliceStaticPublisherInfo::get_status(
+  const DDS::StatusMask mask,
+  void * event)
+{
+  RMW_CHECK_ARGUMENT_FOR_NULL(event, RMW_RET_INVALID_ARGUMENT);
+  switch (mask) {
+    case DDS::LIVELINESS_LOST_STATUS:
+      {
+        DDS::LivelinessLostStatus liveliness_lost;
+        DDS::ReturnCode_t dds_return_code =
+          topic_writer->get_liveliness_lost_status(liveliness_lost);
+        rmw_ret_t from_dds = check_dds_ret_code(dds_return_code);
+        if (from_dds != RMW_RET_OK) {
+          return from_dds;
+        }
+
+        rmw_liveliness_lost_status_t * rmw_liveliness_lost =
+          static_cast<rmw_liveliness_lost_status_t *>(event);
+        rmw_liveliness_lost->total_count = liveliness_lost.total_count;
+        rmw_liveliness_lost->total_count_change = liveliness_lost.total_count_change;
+        break;
+      }
+    case DDS::OFFERED_DEADLINE_MISSED_STATUS:
+      {
+        DDS::OfferedDeadlineMissedStatus offered_deadline_missed;
+        DDS::ReturnCode_t dds_return_code = topic_writer
+          ->get_offered_deadline_missed_status(offered_deadline_missed);
```

Comment:
nitpick: weird place to wrap, might want to look for them, `     ->` might be something to search for.

Reply:
Fixed.

I did a search for ` ->` and it looks like that was the last one.